internal/formatter.go

```diff
@@ -1584,10 +1584,21 @@ func (n *InsertStmtNode) FormatSQL(ctx context.Context) (string, error) {
 		}
 		rows = append(rows, fmt.Sprintf("(%s)", sql))
 	}
-	return fmt.Sprintf("INSERT INTO `%s` (%s) VALUES %s",
+	if len(rows) > 0 {
```

Comment:
Please nil check of `n.node.Query()` before check `n.node.RowList()` like the following.

```go
query := n.node.Query()
if query != nil {
  stmt, err := newNode(query).FormatSQL(ctx)
  if err != nil {
    return "", err
  }
  return fmt.Sprintf("...")
}

rows := []string{}
for _, row := range n.node.RowLis() {

}
```